tools/rdock/rdock.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>
+    <requirements>
+        <requirement type="package" version="2013.1">rDock</requirement>
+    </requirements>
+    <command>
+        <![CDATA[ 
+    	ln -s $inputmol2 inputmol2.mol2 && cat $inputprm | sed "s|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g" | sed "s|REF_MOL.*|REF_MOL ligand.sd|g" > new.prm; 
+        cp -r $__tool_directory__/tools/rdock/data .;
+        mkdir data/ligands; 
+        cp $inputsd data/ligands/ligand.sd; 
+        rbcavity -r new.prm -was > /tmp/rbcavity.log; 
+	rbdock -r new.prm -p dock.prm -n 100 -i $inputsd -o output_docking_out > /tmp/docking_out.log;
+        sdsort -n -f'SCORE' output_docking_out.sd > sorted.sd; 
+        sdrmsd $inputsd sorted.sd > $output_rdock
+    	]]>
+    </command>
+    <inputs>
+        <param name="inputprm" format="prm" type="data" label="Receptor param file (contains active site params)" help="(-r)"/>
+        <param name="inputmol2" format="mol2" type="data" label="Input MOL2 file" />
+        <param name="inputsd" format="sd" type="data" label="Input ligand SD file" help="(-i)"/>

```

Comment:
Can you change `format="sd"` to `format="sdf"`, which is the datatype name used in Galaxy?